api/services/elastigroup/gcp/paths/group-event.yaml

```diff
@@ -6,7 +6,30 @@ get:
     - Elastigroup GCP
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/groupId.yaml"
+    - in: "path"
```

Comment:
@vlad666spot please use the commons params here for groupId, fromDate, toDate.

Reply:
fixed